commercial/app/model/capi/Lookup.scala

```diff
@@ -37,7 +37,7 @@ class Lookup(contentApiClient: ContentApiClient) extends Logging with implicits.
   def latestContentByKeyword(keywordId: String, maxItemCount: Int)(implicit executionContext: ExecutionContext): Future[Seq[ContentType]] = {
     contentApiClient.getResponse(contentApiClient.item(keywordId.stringDecoded, defaultEdition)
       .pageSize(maxItemCount)
-      .showTags("type")
+      .showTags("type,series")
```

Comment:
This line ensures that the response from CAPI shows the tags we need to discover branding information. It changes the query params of the CAPI request:

`commercial/api/capi-multiple.json?k=small-business-network%2Fseries%2Fbusiness-made-simple?show-tags=type,series`

We are still only finding the latest content by keyword, however we just get a few more tags back about each piece of content. The results remain the same, but the series tags are needed if the activeSponsorShip is tied to a series, otherwise `EditionBranding` will not discover it. 😄